.circleci/config.yml

```diff
@@ -13,6 +13,7 @@ commands:
     - run: make style check_license vet staticcheck
   no-style-test:
     steps:
+    - run: go get golang.org/x/sync/errgroup
```

Comment:
Hmm, I *think* this shouldn't be necessary when build, vet, and staticcheck run with Go module support enabled? E.g. https://github.com/prometheus/procfs/blob/master/Makefile.common#L129

But my Go module foo is weak.

Reply:
You're right regarding Go modules but this command is only called for Go 1.10.x. Otherwise I guess that you can simply do `go get -t ./...`.

Reply:
Ah, didn't see that context. But yeah, doing a generate `go get` is much better than hardcoding the dependencies here.

Reply:
Yes, I added this because the Go 1.10 build was broken.

@simonpasquier I'll change this.